app/assets/stylesheets/styles.css

```diff
@@ -0,0 +1,478 @@
+html,
```

Comment:
i get why you extracted this, but _if_ this + the rest of our header content is under 14kb it can be inlined so that it all comes through on the first http round trip, which would improve render performance.

Reply:
@tylerball What do you think about keeping the CSS inline? https://github.com/Shopify/shopify_app/pull/647#discussion_r228277238